account_bank_statement_import_auto_reconcile/models/account_bank_statement_import_auto_reconcile.py

```diff
@@ -49,21 +49,19 @@ def _reconcile_move_line(self, statement_line, move_line_id):
         """Helper to reconcile some move line with a bank statement.
         This will create a move to reconcile with and assigns journal_entry_id
         """
-        move = self.env['account.move'].create(
-            self.env['account.bank.statement']._prepare_move(
-                statement_line,
-                (
-                    statement_line.statement_id.name or statement_line.name
-                ) + "/" + str(statement_line.sequence or '')
-            )
+        move = self.env['account.move'].create({
+            'journal_id': statement_line.statement_id.journal_id.id,
+            'date': statement_line.date,
+            'name': (
+                statement_line.statement_id.name or statement_line.name
+                ) + "/" + str(statement_line.sequence or ''),
+            'ref': statement_line.ref,
+            })
+        move_line_dict = statement_line._prepare_reconciliation_move_line(
+            move,
+            -self.env['account.move.line'].browse(move_line_id).balance,
```

Comment:
I believe this should be `statement_line.amount`. If unsure, debug through a manual reconciliation and see what Odoo does